lib/ansible/modules/cloud/centurylink/clc_loadbalancer.py

```diff
@@ -866,23 +853,16 @@ def define_argument_spec():
         :return: argument spec dictionary
         """
         argument_spec = dict(
-            name=dict(required=True),
-            description=dict(default=None),
-            location=dict(required=True),
-            alias=dict(required=True),
-            port=dict(choices=[80, 443]),
-            method=dict(choices=['leastConnection', 'roundRobin']),
-            persistence=dict(choices=['standard', 'sticky']),
+            name=dict(type='str', required=True),
+            description=dict(type='str'),
+            location=dict(type='str', required=True),
+            alias=dict(type='str', required=True),
+            port=dict(type='str', choices=[80, 443]),
```

Comment:
the default is `type='str'`, so it was `str` before and that is why I agree this is a no change. But I wonder if this shouldn't be a `type='int'`?

Reply:
I agree. I noticed the same thing when editing, however I did not want to change the behaviour of the module in this PR.

One benefit of always specifying the type for parameters is that such issues become more visible. Especially older modules suffer from this.